subsystems/visionsubsystem.py

```diff
@@ -278,11 +278,16 @@ def periodic(self):
         targetAngle = self.trackingModule.getTargetAngle()
         if targetAngle is None:
             SmartDashboard.putBoolean(constants.kTargetPoseArrayKeys.validKey, False)
+            SmartDashboard.putBoolean(
+                constants.kRobotVisionPoseArrayKeys.validKey, False
+            )
         else:
             targetDistance = self.trackingModule.getTargetDistance()
             if targetDistance is None:
                 targetDistance = float("inf")
 
+            targetDistance += constants.kSimTargetUpperHubRadius
```

Comment:
Is this going from hub center? If not, should it be subtracted?

Reply:
the distance to the target's center is the distance to the edge + the radius of the target